<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtdemoClass</class>
 <widget class="QMainWindow" name="qtdemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>915</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MD5计算</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/qtdemo/1.ico</normaloff>:/qtdemo/1.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0">
       <widget class="QPushButton" name="calButton">
        <property name="font">
         <font>
          <family>JetBrains Mono</family>
         </font>
        </property>
        <property name="text">
         <string>计算MD5</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QListWidget" name="leftListWidget"/>
        </item>
        <item row="0" column="1">
         <widget class="QListWidget" name="rightListWidget"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="selectBtn">
        <property name="font">
         <font>
          <family>JetBrains Mono</family>
         </font>
        </property>
        <property name="text">
         <string>选择文件</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>915</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>其他</string>
    </property>
    <addaction name="clearDataAction"/>
    <addaction name="infoAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="clearDataAction">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="infoAction">
   <property name="text">
    <string>Build By thy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtdemo.qrc"/>
 </resources>
 <connections/>
</ui>
